plasmapy/classes/plasma.py

```diff
@@ -89,3 +96,93 @@ def alfven_speed(self):
         B = self.magnetic_field
         rho = self.density
         return np.sqrt(np.sum(B * B, axis=0) / (mu0*rho))
+
+
+class PlasmaBlob():
+    """
+    Class for describing and calculating plasma parameters without
+    spatial/temporal description.
+    """
+    def __init__(self, T_e, n_e, Z=None, particle='p'):
+        """
+        Initialize plasma paramters.
+        The most basic description is composition (ion), temperature,
+        density, and ionization.
+        """
+        self.T_e = T_e
+        self.n_e = n_e
+        self.particle = particle
+        if Z == None:
```

Comment:
I know a certain function you could use here if you wait for #303 :)

Reply:
indeed! This shall be part of the TODO list!